theme/templates/resource-landing-page/subject.html

```diff
@@ -39,25 +39,26 @@ <h3>Subject Keywords</h3>
                         </div>
                     {% endif %}
                 {% endif %}
-
-                <ul id="lst-tags" class="custom-well tag-list">
+                <ul id="lst-tags" class="custom-well tag-list" style="display:none">
                     <li class="small text-muted" v-if="!resKeywords.length">No subject keywords have been added.</li>
                     <li v-else v-for="k in resKeywords">
                         {% if resource_edit_mode %}
-                            <span v-if="resKeywords.length" class="tag">${ k }<span
-                                    @click="removeKeyword('{{ cm.short_id }}', k )"
-                                    class="glyphicon glyphicon-remove-circle icon-remove"></span></span>
+                            <span v-if="resKeywords.length" class="tag">
+                                {% templatetag openvariable %} k {% templatetag closevariable %}<span
+                                @click="removeKeyword('{{ cm.short_id }}', k )"
+                                class="glyphicon glyphicon-remove-circle icon-remove"></span>
+                            </span>
                         {% else %}
                             <a class='tag' target="_blank"
-                               :href="'/search/?q=&selected_facets=subject_exact:' + k">${ k }</a>
+                               :href="'/search/?subject=' + k">{% templatetag openvariable %} k {% templatetag closevariable %}</a>
                         {% endif %}
                     </li>
                 </ul>
             </div>
-            <div>
+            <div id="subj-inline-error" style="display:none">
                 <p class="text-danger small" v-if="showIsDuplicate"> Duplicate. Keyword not added.</p>
```

Comment:
style here as a reminder and easier to trace when cleaning up this band-aid code until the larger PR (already in the works) for RLP cleanup is ready to be discussed